src/stan-users-guide/style-guide.Rmd

```diff
@@ -123,40 +127,40 @@ outside of both loops, and used a total of `N` times.
 
 ## Parentheses and brackets
 
-### Optional parentheses for single-statement blocks {-}
+### Braces for single-statement blocks {-}
 
-Single-statement blocks can be rendered in one of two ways.  The fully
-explicit bracketed way is as follows.
+Single-statement blocks can be rendered in several ways.
+The preferred style is fully bracketed, as follows.
 
 ```
 for (n in 1:N) {
   y[n] ~ normal(mu,1);
 }
 ```
 
-The following statement without brackets has the same effect.
+The following statement without brackets has the same effect,
+but should be avoided.
 
 ```
 for (n in 1:N)
   y[n] ~ normal(mu,1);
 ```
 
-Single-statement blocks can also be written on a single line, as
-in the following example.
+Finally, single-statement blocks can also be written on a single 
```

Comment:
I think we should just eliminate this kind of judgement call and say it should always have braces and newlines.